addons/mail/controllers/discuss.py

```diff
@@ -564,6 +564,17 @@ def audio_worklet_processor(self):
             ]
         )
 
+    @http.route('/mail/emoji/get_data', methods=['GET'], type='http', auth='public', cors="*")
+    def get_emoji_data(self):
+        """ Returns a JS file containing emoji data for emoji picker
+        """
+        return request.make_response(
+            file_open('mail/static/src/models_data/emoji_data.js', 'rb').read(),
```

Comment:
We should probably do something similar to `'/im_livechat/external_lib.<any(css,js):ext>'`